ocs_ci/utility/azure_utils.py

```diff
@@ -315,28 +316,45 @@ def get_disk_state(self, volume_name):
             self.cluster_resource_group, volume_name
         ).disk_state
 
-    def start_vm_instance(self, vm_name):
+    def start_vm_instances(self, vm_names):
         """
-        Start an Azure vm instance
+        Start Azure vm instances
 
         Args:
-            vm_name: Name of azure vm instance
+            vm_names (list): Names of azure vm instances
 
         """
-        result = self.compute_client.virtual_machines.start(
-            self.cluster_resource_group, vm_name
-        )
-        result.wait()
+        for vm_name in vm_names:
+            result = self.compute_client.virtual_machines.start(
+                self.cluster_resource_group, vm_name
+            )
+            result.wait()
 
-    def stop_vm_instance(self, vm_name):
+    def stop_vm_instances(self, vm_names, force=True):
```

Comment:
as discussed, let's set it here to False, to be aligned with Azure API default

Reply:
updated azure_utils.py